crates/orderbook/openapi.yml

```diff
@@ -579,11 +579,16 @@ components:
         Provides the additional data for ethflow orders
       type: object
       properties:
-        is_refunded:
+        refunded_status:
           description: |
-            Provides a flag that shows whether the refunding service has 
-            refunded the order, in case it was not settled onchain
-          type: boolean
+            Describes the status of the refund
+          type: string
+          enum: 
+            [
+              NonRefundable,
+              NotYetRefunded,
+              Refunded,
+            ]
```

Comment:
[only good to have] Not for launch, but it would be nice to add the `cancelTx` and `createTx` to show in the explorer.